src/components/game/Game.play.test.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Enzyme from 'enzyme';
 import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
+import { waitFor } from '@testing-library/react';
```

Comment:
Codacy has a fix for the issue: [Replace `{·waitFor·}·from·'@testing-library/react` with `Enzyme·from·'enzyme`](https://app.codacy.com/gh/msaperst/number-munchers/pullRequest?prid=8124485)
```suggestion
import Enzyme from 'enzyme';
```